ozaria/engine/cinematic/CinematicLankBoss.js

```diff
@@ -480,7 +480,21 @@ const createThang = thang => {
     scaleFactorX: 1,
     scaleFactorY: 1,
     action: 'idle',
-    rotation: LEFT
+    rotation: LEFT,
+    // This method is required by the Lank to support customization
+    getLankOptions: function () {
+      // TODO: Make this only applied to hero character instead of anything customizable.
+      let options = { colorConfig: {} }
+      const playerTints = (me.get('ozariaHeroConfig') || {}).tints || []
+      playerTints.forEach((tint) => {
+        const colorGroups = (tint.colorGroups || {})
+        for (const key in colorGroups) {
```

Comment:
It seems like this can also be accomplished with 

```
const colorGroups = (tint.colorGroups || {})
options.colorConfig = colorGroups
```

Reply:
After conversation IRL this could either use spread operator or lodash merge.  Not a critical change though.

Reply:
@jmif Appreciate you pointing this out. I cleaned up the code a bit. Used the lodash merge operator as spread isn't supported just yet.